src/services/SpeciesService.ts

```diff
@@ -279,6 +281,63 @@ const getSpeciesNames = async (search: string): Promise<SpeciesNamesResponse> =>
   return response;
 };
 
+/**
+ * Search species for selectors
+ */
+const suggestSpecies = async (organizationId: number, query: string): Promise<SuggestedSpecies[] | null> => {
+  const params: SearchRequestPayload = {
+    prefix: 'species',
+    fields: ['scientificName', 'commonName', 'familyName', 'id'],
+    search: {
+      operation: 'and',
+      children: [
+        {
+          operation: 'field',
+          field: 'organization_id',
+          type: 'Exact',
+          values: [organizationId],
+        },
+      ],
+    },
+    count: 0,
+  };
+
+  if (query) {
+    const searchValueChildren: FieldNodePayload[] = [];
+
+    const nameNode: FieldNodePayload = {
+      operation: 'field',
+      field: 'scientificName',
+      type: 'Fuzzy',
+      values: [query],
+    };
+    searchValueChildren.push(nameNode);
+
+    const commonNameNode: FieldNodePayload = {
+      operation: 'field',
+      field: 'commonName',
+      type: 'Fuzzy',
+      values: [query],
+    };
+    searchValueChildren.push(commonNameNode);
+
+    const familyNode: FieldNodePayload = {
```

Comment:
Why are we adding family? If a species name is totally different from its family (not sure if that happens) it will still suggest that species name when writing a totally different thing in the input